yarn.lock

```diff
@@ -6858,9 +6858,9 @@ jest-worker@^27.4.5:
     merge-stream "^2.0.0"
     supports-color "^8.0.0"
 
-jpeg-js@0.4.3:
-  version "0.4.3"
-  resolved "https://registry.yarnpkg.com/jpeg-js/-/jpeg-js-0.4.3.tgz#6158e09f1983ad773813704be80680550eff977b"
+jpeg-js@0.4.4:
```

Comment:
Did you run yarn after updating this? It looks like playwright-core is specifying 0.4.3 directly 

`jpeg-js "0.4.3"`

Which means this is going to get overwritten when yarn is ran and fail if ran with a frozen lock file.